pkg/tfbridge/schema_test.go

```diff
@@ -2588,6 +2589,200 @@ func TestExtractDefaultIntegerInputs(t *testing.T) {
 	assert.Equal(t, expected, ins)
 }
 
+func TestExtractSchemaInputsNestedMaxItemsOne(t *testing.T) {
```

Comment:
Looks good! Trust these test cases discriminate the bug.

Reply:
I checked. Both fail in different ways on the old code.